lib/rubocop/config_loader.rb

```diff
@@ -60,15 +60,20 @@ def merge(base_hash, derived_hash)
 
       def base_configs(path, inherit_from)
         configs = Array(inherit_from).compact.map do |f|
-          f = File.expand_path(f, File.dirname(path))
+          if f =~ URI.regexp
```

Comment:
This might match unexpectedly. I typically use the accepted answer to http://stackoverflow.com/questions/1805761/check-if-url-is-valid-ruby
```ruby
f =~ /\A#{URI::regexp(['http', 'https'])}\z/
```

I'm on a Mac, so I'm not sure about this example:

```ruby
f = 'c:\\\\foo\\bar.yml'
!!(f =~ URI.regexp) #=> true
```